docs/basic-docs/atomicdex/atomicdex-api.md

```diff
@@ -581,6 +581,144 @@ curl --url "http://127.0.0.1:7783" --data "{\"userpass\":\"$userpass\",\"method\
+
+::: tip
+
+Use the pubkey without prefix for this method input. E.g. if pubkey is `022cd3021a2197361fb70b862c412bc8e44cff6951fa1de45ceabfdd9b4c520420` you should submit `2cd3021a2197361fb70b862c412bc8e44cff6951fa1de45ceabfdd9b4c520420`.
+
+:::
+
+#### Arguments
+
+| Structure       | Type                       | Description                                                                   |
+| --------------- | -------------------------- | ----------------------------------------------------------------------------- |
+| pubkey          | string                     | the pubkey to ban                                                             |
+| reason          | string                     | the reason of banning                                                         |
+
+#### Response
+
+| Structure              | Type     | Description                                                                                                                                                                                                     |
+| ---------------------- | -------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| result                 | string   | whether the ban was successful                                                                                                                                                                                  |
+                                                                                                                                                               
+#### :pushpin: Examples
+
+#### Command
+
+```bash
+curl --url "http://127.0.0.1:7783" --data '{
+  "userpass":"'$userpass'",
+  "method": "ban_pubkey",
+  "pubkey": "2cd3021a2197361fb70b862c412bc8e44cff6951fa1de45ceabfdd9b4c520420",
+  "reason": "test",
+}'
+```
+
+<div style="margin-top: 0.5rem;">
+
+<collapse-text hidden title="Response">
+
+#### Response (success)
+
+```json
+{
+  "result": "success"
+}
+```
+
+</collapse-text>
+
+</div>
+
+## best\_orders
+
+**best_orders coin action volume**
+
+The `best_orders` method returns the best price orders that can fill the volume for all existing pairs with selected coin.
+
+#### Arguments
+
+| Structure       | Type                       | Description                                                                   |
+| --------------- | -------------------------- | ----------------------------------------------------------------------------- |
+| coin            | string                     | the ticker of the coin to get best orders                                     |
+| action          | string                     | whether to `buy` or `sel` the selected coin                                   |
```

Comment:
is `sel` as written here correct, or should it be `sell`? cc: @sergeyboyko0791

Reply:
I guess it's a typo

Reply:
It's typo indeed, will fix shortly :slightly_smiling_face: